src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -313,9 +317,9 @@ private async Task<IEnumerable<RazorCodeAction>> GetRazorCodeActionsAsync(RazorC
             return codeActions;
         }
 
-        private static HashSet<string> GetAllAvailableCodeActionNames()
+        private static HashSet<string?> GetAllAvailableCodeActionNames()
         {
-            var availableCodeActionNames = new HashSet<string>();
+            var availableCodeActionNames = new HashSet<string?>();
```

Comment:
We should just filter for nulls when creating the hashset, there is presumably unlikely to ever actually be any